utils/check_repo.py

```diff
@@ -73,6 +73,8 @@
     "TimeSeriesTransformerDecoder",  # Building part of bigger (tested) model.
     "InformerEncoder",  # Building part of bigger (tested) model.
     "InformerDecoder",  # Building part of bigger (tested) model.
+    "AutoformerEncoder",  # Building part of bigger (tested) model.
```

Comment:
Currently it's grouped with the other time-series models:

```
"TimeSeriesTransformerEncoder",  # Building part of bigger (tested) model.
"TimeSeriesTransformerDecoder",  # Building part of bigger (tested) model.
"InformerEncoder",  # Building part of bigger (tested) model.
"InformerDecoder",  # Building part of bigger (tested) model.
"AutoformerEncoder",  # Building part of bigger (tested) model.
"AutoformerDecoder",  # Building part of bigger (tested) model.
```

Re-top it will break this convention, what do you think?

Reply:
Makes sense :) 👍